pkg/http/api.go

```diff
@@ -302,6 +305,15 @@ func (a *api) onOutputBindingMessage(reqCtx *fasthttp.RequestCtx) {
 		return
 	}
 
+	// pass the trace context to output binding in metadata
+	if span := diag_utils.SpanFromContext(reqCtx); span != nil {
+		sc := span.SpanContext()
+		metadata := map[string]string{}
+		metadata[traceparentHeader] = diag.SpanContextToW3CString(sc)
+		metadata[tracestateHeader] = diag.TraceStateToW3CString(sc)
+		req.Metadata = metadata
+	}
+
```

Comment:
Can you add test? at least unit-test.

Reply:
The unit test for this method with tracing is there and this code path will be taken care of this test ..as no failure observed.

https://github.com/dapr/dapr/blob/613398ca4cf269741dd0957425adb01b0f965aec/pkg/http/api_test.go#L113